cl_dll/in_defs.h

```diff
@@ -17,9 +17,7 @@
 #define	ROLL	2 
 
 #ifdef _WIN32
-#define HSPRITE HSPRITE_win32
```

Comment:
I have noticed it while checking latest commits. Since HSPRITE changed to SpriteHandle_t, it is now ok to add that definition.